components/script/dom/webidls/FormDataEvent.webidl

```diff
@@ -0,0 +1,14 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at https://mozilla.org/MPL/2.0/. */
+
+// https://html.spec.whatwg.org/multipage/#the-formdataevent-interface
+[Exposed=Window,
+ Constructor(DOMString type, optional FormDataEventInit eventInitDict)]
+interface FormDataEvent : Event {
+  readonly attribute FormData formData;
+};
+
+dictionary FormDataEventInit : EventInit {
+  /*required*/ FormData formData;
```

Comment:
What happens if we uncomment this? If there's an error, do we have an issue filed for it, and can we add a comment?

Reply:
It will fail with this compilation error. I didn't find a corresponding issue. I'll try to read the webidl codegen to see what the root cause is later.

```rust
error[E0599]: no function or associated item named `empty` found for type `dom::bindings::codegen::Bindings::FormDataEventBinding::FormDataEventInit` in the current scope
   --> /Users/cybai/codespace/mozilla/servo/target/debug/build/script-59d4418593395066/out/Bindings/FormDataEventBinding.rs:857:95
    |
259 | pub struct FormDataEventInit {
    | ---------------------------- function or associated item `empty` not found for this
...
857 |             crate::dom::bindings::codegen::Bindings::FormDataEventBinding::FormDataEventInit::empty()
    |             ----------------------------------------------------------------------------------^^^^^
    |             |
    |             function or associated item not found in `dom::bindings::codegen::Bindings::FormDataEventBinding::FormDataEventInit`
```

Reply:
Ok, it is clear to me why this happens now. Filed https://github.com/servo/servo/issues/22670 about it.

Reply:
Thanks a lot! After reading the issue, I can understand why it failed to compile with `required` :)